package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Simple reusable React error boundary component",
   "author": "Brian Vaughn <brian.david.vaughn@gmail.com>",
   "license": "MIT",
+  "packageManager": "pnpm@8.6.10",
```

Comment:
declare packageManager version like [pnpm's package.json this code line](https://github.com/pnpm/pnpm/blob/main/package.json#L66C6-L66C6) too